### DISCUSSION TITLE
What happened to the old organization?

### DISCUSSION THREAD

**Discussion by dogtopus** [OpenVizslaTNG/ov_ftdi#11] [General]
@laf0rge Just wondering why the old organization (@openvizsla) was abandoned? The description on this organization mentions that the old one "became unable to recover". What does that mean exactly? Is the credential of the old organization got lost somehow?

**Comment by laf0rge** [comment]
@openvizsla is not a github organization but was the personal account of one of the original developers who passed away in 2014. It might be that at the time it was created, github didn't even have "organizations" yet, or that back then it was not known that they exist.

@tmbinc has attempted to recover via github, who allegedly have a lengthy and difficult process for those situation, but eventually he gave up.

The new project is now a proper organization with multiple users having admin privileges. preventing a similar single point of failure.
